README.md

```diff
@@ -150,7 +150,9 @@ selinux::boolean { 'puppetagent_manage_all_files': }
 ```
 BEAKER_debug=yes BEAKER_set="centos-6-x64" PUPPET_INSTALL_TYPE="agent" bundle exec rake beaker &&
 BEAKER_debug=yes BEAKER_set="centos-7-x64" PUPPET_INSTALL_TYPE="agent" bundle exec rake beaker &&
-BEAKER_debug=yes BEAKER_set="fedora-25-x64" PUPPET_INSTALL_TYPE="agent" bundle exec rake beaker
+BEAKER_debug=yes BEAKER_set="fedora-25-x64" PUPPET_INSTALL_TYPE="agent" bundle exec rake beaker &&
+BEAKER_debug=yes BEAKER_set="fedora-26-x64" PUPPET_INSTALL_TYPE="agent" bundle exec rake beaker &&
+BEAKER_debug=yes BEAKER_set="fedora-27-x64" PUPPET_INSTALL_TYPE="agent" bundle exec rake beaker
```

Comment:
Maybe this should be a for loop

Reply:
I ususally just copy paste one after another or one specific platform to run. with a for loop copy paste would be more error prone. ;-)

Reply:
https://github.com/voxpupuli/modulesync_config/pull/422 would make it easier since it reads it based on the metadata.